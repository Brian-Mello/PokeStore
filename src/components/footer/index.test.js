import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';

describe('testando integridade do Footer component', () => {
    it(' Footer snapshot', () => {
        const footer = renderer.create(<Footer/>).toJSON();
        expect(footer).toMatchSnapshot();
    });
}); 