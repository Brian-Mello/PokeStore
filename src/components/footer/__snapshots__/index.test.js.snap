// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando integridade do Footer component  Footer snapshot 1`] = `
<header
  className="sc-AxjAm cLFrOt"
>
  <p
    className="sc-AxirZ bIKXRc"
  >
    Copyright Â© 
    2020
     
    <b>
      Brian Mello design Limited.
    </b>
  </p>
</header>
`;
